### PR TITLE
Fixes + goto on mouse left

### PR DESCRIPTION
**Pull request by Earu**
Fixed typos (hello techbot), added goto when clicking on a player line with left click, removed end and start useless anims

### CODE REVIEW

File: yscoreboard/lua/autorun/client/yscoreboard.lua [hunk 1]
```diff
@@ -520,16 +488,16 @@ timer.Simple( 1.5, function()
...
      
     end
  
     function GAMEMODE:ScoreboardHide()
      
         if ( IsValid( w_Scoreboard ) ) then
             w_Scoreboard:SetMouseInputEnabled( false )
-            timer.Create( "w_Scoreboard_Hide", 0.1, 1, function() if ( IsValid( w_Scoreboard ) ) then w_Scoreboard:Hide() end end )
-            w_Scoreboard:EndAnim( 0.01 )
+            if ( IsValid( w_Scoreboard ) ) then
```
Comment (Techbot121) [inline yscoreboard/lua/autorun/client/yscoreboard.lua]: why did you do the same if statement twice?

**Review by Techbot121 — CHANGES_REQUESTED** [review]

File: yscoreboard/lua/autorun/client/yscoreboard.lua (hunk 1 above)
Comment (Earu) [reply yscoreboard/lua/autorun/client/yscoreboard.lua]: oops

File: yscoreboard/lua/autorun/client/yscoreboard.lua [hunk 2]
```diff
@@ -307,10 +303,10 @@ local PLAYER_LINE = {
             end
             if aowl ~= nil then
                 local SubAdmin = self.Menu:AddSubMenu("Staff")
-                    SubAdmin:AddOption( "Bring",function() LocalPlayer():ConCommand( "awol bring "..self.Player:Nick() ) end)
-                    SubAdmin:AddOption( "Kick",function() cinputs( "aowl kick "..self.Player:Nick() ) end)
-                    SubAdmin:AddOption( "Ban",function() cinputs( "aowl ban "..self.Player:Nick() ) end)
-                    SubAdmin:AddOption( "Reconnect",function() LocalPlayer():ConCommand( [[awol retry]]) end)
+                    SubAdmin:AddOption( "Bring",function() RunConsoleCommand("aowl","bring",self.Player:Nick() ) end)
+                    SubAdmin:AddOption( "Kick",function() cinputs( "aowl kick "..self.Player:Nick(),3 ) end)
```
Comment (Techbot121) [inline yscoreboard/lua/autorun/client/yscoreboard.lua]: the cinputs function is still there

**Review by Techbot121 — APPROVED** [review]
ok I think that was all that I saw while quickly looking through it.

**Pull request merged**